examples/templating/htmlTemplating/index.html

```diff
@@ -0,0 +1,40 @@
+<html lang="en">
```

Comment:
Should we rename the `htmlTemplating` directory into `html`? The term "templating" is redundant with the parent folder.